modules/odc-installing-helm-charts.adoc

```diff
@@ -5,26 +5,19 @@
 [id="odc-installing-helm-charts_{context}"]
 = Installing Helm charts
 
-You can use the *Developer* perspective to create Helm releases from the Helm charts provided in the *Developer Catalog*.
-
-.Prerequisites
-
-* You have logged in to the web console and have switched to the *Developer* perspective.
+You can use either the *Developer* perspective or the CLI to create Helm releases and see them in the Developer perspective of the web console.
 
 .Procedure
+To create Helm releases from the Helm charts provided in the *Developer Catalog*:
 
-. In the *Developer* perspective, navigate to the *Add* view and select the *From Catalog* option to see the *Developer Catalog*.
-. In the filters listed on the left, under *Type*, select the *Helm Charts* filter to see the available Helm charts.
+. In the *Developer* perspective, navigate to the *Add* view and select the *Helm Chart* option to see all the Helm Charts in the *Developer Catalog*.
+. Select a chart to see the description, README, and other details of the Helm chart and click *Install Helm Chart*.
```

Comment:
Can we use **its** here, to avoid repetition? somthing like -  Select a chart to see its description, README...and click ..

Reply:
Good call, a bit worried about anthropomorphism, so I have restructured it a bit, hope that helps.